po/de.po

```diff
@@ -25,25 +25,86 @@ msgid "Extension Manager"
-msgstr "Ein Programm zum Verwalten von GNOME Shell Erweiterungen "
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:9
+msgid "Matthew Jakeman"
+msgstr ""
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:12
+msgid "Features:"
+msgstr "Funktionen:"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:18
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:14
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:24
 msgid "Browse extensions.gnome.org right inside the app"
-msgstr "Durchstöbern Sie extensions.gnome.org direkt in der App "
+msgstr "Durchstöbern Sie extensions.gnome.org direkt in der App"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:22
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:15
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:28
 msgid "Manage the extensions you already have installed"
 msgstr "Verwalten Sie bereits installierte Erweiterungen"
 
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:20
+msgid "A utility for managing GNOME Shell Extensions"
+msgstr "Ein Programm zum Verwalten von GNOME Shell Erweiterungen"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:43
+msgid "The first feature update to extension-manager. Highlights include:"
+msgstr "Das erste Funktionsupdate für Erweiterungs Manager. Höhepunkte sind:"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:45
+msgid "Dark theme and support for overriding the system colour scheme"
+msgstr ""
+"Dunkles Theme und unterstützung für das Überschreiben des System Farbschemas"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:46
+msgid "Shows screenshots in extension listings"
+msgstr "Zeige Bildschirmfotos in Erweitungs-Eintrag"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:47
+msgid "Shell version compatibility check"
+msgstr "Shell-Versions Kompatibilitätsprüfung"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:48
+msgid "Localised into seven languages"
+msgstr "Übersetzt in sieben Sprachen"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:49
+msgid "Sort search results"
+msgstr "Suchergebnisse sortieren"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:50
+msgid "Global extension on/off toggle"
+msgstr "Globaler Erweiterungs an/aus Schalter"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:51
+msgid "Displays user and system extensions separately"
+msgstr "Zeige Nutzer- und System-Erweiterungen getrennt an"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:52
+msgid "Performance and stability"
+msgstr "Leistung und Stabilität"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:11
+msgid "Colour Scheme Preference"
+msgstr "Farbschema Einstellungen"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:12
+msgid ""
+"Whether to follow the system colour scheme or force either light or dark "
+"mode."
+msgstr ""
+"Ob es wie das System-Farbschema oder Dunkler oder Heller Schema erzwungen "
```

Comment:
This is no German sentence. What about:

Ob das System-Farbschema oder der helle oder dunkle Modus bevorzugt werden soll.

Reply:
Oh overlooked that.